src/components/Layout/ScrollableParent.tsx

```diff
@@ -63,7 +63,15 @@ export function ScrollableParent(props: PropsWithChildren<ScrollableParentContex
       <Tag css={{ ...Css.mh0.mw0.fg1.df.fdc.$, ...otherXss }}>
         {/* Use `overflow: overlay` to prevent scrollbar from pushing content in when rendered. This is only supported in Webkit browsers, so we also keep `overflow: auto` as a fallback for other browsers.*/}
         {/* `overlay` needs to be applied using `addIn` otherwise the two `overflow` properties will be merged. */}
-        <div css={Css.pl(context.pl).pr(context.pr).if(!hasScrollableContent).h100.overflowAuto.$}>{children}</div>
+        <div
+          css={{
+            ...Css.pl(context.pl).pr(context.pr).if(!hasScrollableContent).h100.overflowAuto.$,
+            // Not using `Css.addIn` as it is added regardless of the `Css.if()` conditional
+            ...(!hasScrollableContent ? { "&:after": Css.contentEmpty.db.h2.$ } : undefined),
```

Comment:
I wonder if making a shared variable for the contentEmpty.db.h2 would be too cute; it would give a natural spot for a shared comment.